gosling/api.py

```diff
@@ -43,10 +34,22 @@ class _PropertiesMixen:
     def properties(self, **kwargs):
         copy = self.copy()
         for key, value in kwargs.items():
-            copy[key] = value
+            setattr(copy, key, value)
```

Comment:
Just curious: Does the use of `setattr()` affect any differences compared with the previous one (i.e., `copy[key] = value`)?

Reply:
In practice I don't think there is a difference because the `copy` class inherits from `SchemaBase` which defines `__setitem__` as an alias for `setattr`. I was just looking through the altair source and saw this pattern.